cmd/repo-updater/repos/syncer.go

```diff
@@ -162,10 +236,35 @@ func (s *Syncer) Sync(ctx context.Context) (err error) {
 	return nil
 }
 
+func calcSyncInterval(now time.Time, lastSync time.Time, minSyncInterval time.Duration, diff Diff) time.Duration {
+	const maxSyncInterval = 8 * time.Hour
+
+	// Special case, we've never synced
+	if lastSync.IsZero() {
+		return minSyncInterval
+	}
+
+	log15.Info("Diffs", "added", len(diff.Added), "deleted", len(diff.Deleted), "modified", len(diff.Modified))
```

Comment:
Pretty sure this could be a DEBUG :D 
```suggestion
	log15.Debug("Diffs", "added", len(diff.Added), "deleted", len(diff.Deleted), "modified", len(diff.Modified))
```

Reply:
I think this one is a leftover of our long debugging sessions